api/hello.js

```diff
@@ -0,0 +1,5 @@
+export default function handler(req, res) {
```

Comment:
This file is specific to Vercel. It's an Edge function that acts like an API endpoint in your vercel deployment.

Reply:
The vercel.json file you'll see has the corn specs. This API endpoint will be called every 14 minutes so Render will never be inactive therefore no cold start problem.